app/models/fiber_channel_address.rb

```diff
@@ -0,0 +1,5 @@
+class FiberChannelAddress < SanAddress
+  def get_address_info
```

Comment:
I don't love the "one method to return everything you need" style.  I wonder if we can follow the `self.display_name(number = 1)` pattern with most of the other models and then just have `address` or `address_info` return the specific address for the different type of addresses (wwpn or iqn)

Reply:
@skateman what do you think on the display_name?

Reply:
What is this thing returning? I need a little bit more context :thinking:

Reply:
Depending on the type of address it is returning the name of the attribute, so wwpn for fibre channel and iqn for iSCSI

Reply:
Yeah, this seems like it's heavily tied to the summary screens specification, I don't think this should be structured in the model at all. Also rubocop is probably complaining about the getter method. 

@agrare are you suggesting here to split this up to two methods where one will be the `display_name`?

Reply:
@skateman correct

Reply:
@agrare @skateman I thought that the get_address method will be helpful in the future once other provider will also use the san_address model, but we can remove it completely like @skateman suggested and keep all the logic in the UI-classic, e.g :

def textual_group_san_addresses
    port_type = {
      "IscsiAddress"        => "IQN",
      "FiberChannelAddress" => "WWPN",
      "NvmeAddress"         => "WWPN"
    }

  san_addresses_values = @record.san_addresses.map do |san_address|
    san_address_type = port_type[san_address.type]
    [san_address_type, san_address.send(san_address_type.downcase)]
  end
...

-----------

we can also get only the address info.
in the core:
  def address_value
    iqn
  end

in the ui-classic:
def textual_group_san_addresses
  port_type = {
    "IscsiAddress"        => "IQN",
    "FiberChannelAddress" => "WWPN",
    "NvmeAddress"         => "WWPN"
  }

  san_addresses_values = @record.san_addresses.map do |san_address|
    [port_type[san_address.type], san_address.address_info]
  end
...

I'm not sure why we need the display_name. @gregoryb-at-ibm any thoughts on this subject?

Reply:
@llivne ,
What @agrare and @skateman suggest is functional equivalent to your last suggestion.
Please comply with MIQ established practices and add the display_name method as request

```
san_addresses_values = @record.san_addresses.map do |san_address|
  [san_address.class.display_name, san_address.address_value]
end
```

Reply:
@agrare @skateman @gregoryb-at-ibm I changed the code to use display_name (also in the ui-classic). please take a look.